src/utils/InputWrapper.js

```diff
@@ -76,7 +76,10 @@ InputWrapper.textProp = 'value';
  */
 InputWrapper.focus = function() {
     if (InputWrapper.hiddenInput) {
-        InputWrapper.hiddenInput.focus();
+        // the hidden input element only gets focused when the focus function is called within a timeout function
+        window.setTimeout(function () {
+            InputWrapper.hiddenInput.focus();
+        }, 0);
```

Comment:
are we sure that this  "hack" works reliable in all browsers?  Or should we make loop and check every few ms if the element is focused and try again until it works?